src/signals/settings/users/containers/Overview/hooks/useFetchUsers.js

```diff
@@ -36,18 +36,18 @@ const useFetchUsers = ({ page, filters } = {}) => {
 
       const pageParams = [page && `page=${page}`, `page_size=${PAGE_SIZE}`];
 
+
       const filterParams = Object.entries(filters || {})
-        .filter(([, value]) => Boolean(value))
+        .filter(([, value]) => !['*'].includes(value))
```

Comment:
Why not
```jsx
.filter(([, value]) => value !== '*')
```
Bit easier to read, wouldn't you agree?

Reply:
The reason I used the includes is that I filtered for multiple values in the first place, indeed not needed.